bluepysnap/morph.py

```diff
@@ -28,11 +28,16 @@
 from bluepysnap.exceptions import BluepySnapError
 from bluepysnap.utils import is_node_id
 
+EXTENSIONS_MAPPING = {
+    'asc': 'neurolucida-asc',
+    'h5': 'h5v1',
+}
+
 
 class MorphHelper:
     """Collection of morphology-related methods."""
 
-    def __init__(self, morph_dir, population):
+    def __init__(self, morph_dir, population, alternate_morph_dir=None):
```

Comment:
`alternate_morph_dir` should be added to the docstring.
Considering that it's a collection (a dict) and not a single string, a plural name `alternate_morph_dirs` would be clearer.
Or it can be called `alternate_morphologies` that's the same name used in the configuration.

Reply:
I agree. `alternate_morphologies` is best here. I wonder why I decided to go w/ this?

Reply:
fixed